packages/hash/api/src/graphql/resolvers/pages/setParentPage.ts

```diff
@@ -0,0 +1,75 @@
+import { ApolloError } from "apollo-server-express";
+import { MutationSetParentPageArgs, Resolver } from "../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../context";
+import { Entity, Page, UnresolvedGQLEntity } from "../../../model";
+import { topologicalSort } from "../../../util";
+
+export const setParentPage: Resolver<
+  Promise<UnresolvedGQLEntity>,
+  {},
+  LoggedInGraphQLContext,
+  MutationSetParentPageArgs
+> = async (_, { accountId, pageId, parentPageId }, { dataSources, user }) => {
+  return await dataSources.db.transaction(async (client) => {
+    const pageEntity = await Page.getAccountPageWithParents(client, {
+      accountId,
+      entityId: pageId,
+    });
+
+    const parentPageEntity = await Entity.getEntityLatestVersion(client, {
+      accountId,
+      entityId: parentPageId,
+    });
+
+    if (!pageEntity || !parentPageEntity) {
+      const notFoundId = pageEntity?.entityId
+        ? `'${parentPageId}'`
+        : `'${pageId}' ${
+            parentPageEntity?.entityId
+              ? ""
+              : `nor parent page with entityId = '${parentPageId}`
```

Comment:
```suggestion
              : ` nor parent page with entityId = '${parentPageId}'`
```

Reply:
https://github.com/hashintel/hash/blob/66b928cec19baa13f8bac0353eb5a545a0ad7d22/packages/hash/api/src/graphql/resolvers/pages/setParentPage.ts#L27 the space is added before the interpolation. But I think i would prefer for it to be here instead, actually. I'll change it :)

Reply:
Oh yes, my mistake, apologies